__tests__/main.test.js

```diff
@@ -1,25 +1,57 @@
-import Car from '../src/main';
+import { isInputValid } from '../src/utils';
 
-describe('자동차 클래스 테스트', () => {
-  let car = null;
+describe('콘솔 입력에 대한 유효성 검사 동작 테스트', () => {
+  test('사용자 입력란에 "아반떼"을 입력하면 유효성 검사는 통과이다.', () => {
```

Comment:
테스트는 단언형으로 작성되었을 때 더 읽기 좋은거 같아요 ㅎㅎ
```suggestion
  test('사용자 입력란에 "아반떼"을 입력하면 유효성 검사를 통과한다.', () => {
```